ledger/simulation/simulator.go

```diff
@@ -190,13 +203,14 @@ func (s Simulator) Simulate(txgroup []transactions.SignedTxn) (*ledgercore.Valid
 	}
 	nextBlock := bookkeeping.MakeBlock(prevBlockHdr)
 	hdr := nextBlock.BlockHeader
+	simulatorTracer := makeTracer()
```

Comment:
This seems to be installing a NullEavlTracer.  I don't quite understand why, but regardless, wouldn't `nil` be faster/better?

Reply:
Leaving a note as a test of my understanding and obviously defer to @jasonpaulos here for an authoritative response:
* My sense is `simulatorTracer` is a placeholder for later installing a tracer with non-null behavior.  
* I'm guessing it was left as shown (rather than `nil`) to make it more obvious a change can be made _here_.

Reply:
that makes sense